index.html

```diff
@@ -4743,13 +4615,13 @@ <h3>Verify Securing Mechanism</h3>
 Set <var>securedDocument</var> to the result of a JSON parsing function that
```

Comment:
GitHub does not let me comment on text that has not changed, so I put it here; it concerns the 3.1) step. That text uses the terminology "by utilizing the `proof`.`type` property". However, these operations happen on the result of bridging the JSON bytes onto Infra, so (I presume) we are talking about considering an infra _map_ at this step. If so, the terminology is "key" and not "property".

Reply:
Not all web spec authors agree with me here (e.g. I think the Infra editors disagree), but I don't think it's super important to always use the right terminology for the things inside maps, structs, lists, objects, etc. The important thing is that we avoid any possibility for implementers to understand different things from our text, and I haven't been able to come up with a misinterpretation based on this choice of words.

That said, the other words here are somewhat ambiguous. `proof.type` in what object? It'll be _securedDocument_["`proof`"]["`type`"] (or _securedDocument_.proof.type to be less formal)., but _securedDocument_ hasn't been initialized yet. So the text should probably move below its initialization ... except that https://github.com/w3c/vc-data-integrity/pull/227 teaches vc-data-integrity to do this dispatch based on _inputBytes_, so it's probably not worth changing anything in this PR.

Reply:
Yes, waiting on https://github.com/w3c/vc-data-integrity/pull/227 to settle before coming back here and fixing the language. This PR was just to get INFRA integrated in a basic fashion, we'll tighten things up in another pass, after https://github.com/w3c/vc-data-integrity/pull/227 has landed.